{
  "config": {
    "flow_title": "Synology DSM {name} ({host})",
    "step": {
      "user": {
        "title": "Synology DSM",
        "data": {
          "host": "[%key:common::config_flow::data::host%]",
          "port": "[%key:common::config_flow::data::port%]",
          "ssl": "[%key:common::config_flow::data::ssl%]",
          "verify_ssl": "[%key:common::config_flow::data::verify_ssl%]",
          "username": "[%key:common::config_flow::data::username%]",
          "password": "[%key:common::config_flow::data::password%]"
        }
      },
      "2sa": {
        "title": "Synology DSM: two-step authentication",
        "data": {
          "otp_code": "Code"
        }
      },
      "link": {
        "title": "Synology DSM",
        "description": "Do you want to setup {name} ({host})?",
        "data": {
          "ssl": "[%key:common::config_flow::data::ssl%]",
          "verify_ssl": "[%key:common::config_flow::data::verify_ssl%]",
          "username": "[%key:common::config_flow::data::username%]",
          "password": "[%key:common::config_flow::data::password%]",
          "port": "[%key:common::config_flow::data::port%]"
        }
      }
    },
    "error": {
      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
      "missing_data": "Missing data: please retry later or an other configuration",
      "otp_failed": "Two-step authentication failed, retry with a new pass code",
      "unknown": "[%key:common::config_flow::error::unknown%]"
    },
    "abort": {
      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
    }
  },
  "options": {
    "step": {
      "init": {
        "data": {
          "scan_interval": "Minutes between scans",
          "timeout": "Timeout (seconds)"
        }
      }
    }
  }
}
